client/app/scripts/superdesk-search/views/item-search.html

```diff
@@ -1,41 +1,27 @@
 <form>
     <fieldset ng-if="context">
-        <label> 
-            <input type="radio" value="local" ng-model="repo.search"> 
-            In Superdesk
-        </label>
-        <div ng-show="repo.search == 'local'">
-            <div class="field">
-                <label for="ingest-collection">
-                    {{:: 'Ingest' | translate}}
-                    <input type="checkbox" id="ingest-collection" ng-model="repo.ingest" ng-checked="repo.ingest" tabindex="4">
-                </label>
-            </div>
-            <div class="field">
-                <label for="archive-collection">
-                    {{:: 'Production' | translate}}
-                    <input type="checkbox" id="archive-collection" ng-model="repo.archive" ng-checked="repo.archive" tabindex="5">
-                </label>
-            </div>
-            <div class="field">
-                <label for="published-collection">
-                    {{:: 'Published' | translate}}
-                    <input type="checkbox" id="published-collection" ng-model="repo.published" ng-checked="repo.published" tabindex="6">
-                </label>
-            </div>
-            <div class="field">
-                <label for="archived-collection">
-                    {{:: 'Archived' | translate}}
-                    <input type="checkbox" id="archived-collection" ng-model="repo.archived" ng-checked="repo.archived" tabindex="7">
-                </label>
-            </div>
+        <div class="field">
+            <label for="ingest-collection">
+                {{:: 'Ingest' | translate}}
+                <input type="checkbox" id="ingest-collection" ng-model="repo.ingest" ng-checked="repo.ingest">
+            </label>
         </div>
-    </fieldset>
-    <fieldset ng-if="context">
-        <div ng-repeat="provider in providers">
```

Comment:
The Image Search Provider isn't configured in master.dev and that's the reason it's not visible. But we need that option.